ci/recipe/meta.yaml

```diff
@@ -24,12 +23,19 @@ requirements:
     - pandas >=0.25.3
     - scikit-learn
     - scikit-bio
-    - q2-types {{ release }}.*
-    - q2-longitudinal {{ release }}.*
-    - q2-feature-classifier {{ release }}.*
-    - qiime2 {{ release }}.*
+    - q2-types {{ qiime2_epoch }}.*
+    - q2-longitudinal {{ qiime2_epoch }}.*
+    - q2-feature-classifier {{ qiime2_epoch }}.*
+    - qiime2 {{ qiime2_epoch }}.*
```

Comment:
This `qiime2_epoch` variable is defined here:

https://github.com/qiime2/package-integration/blob/main/2021.10/tested/conda_build_config.yaml#L61-L62